specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-02-01-preview/serviceGroups.json

```diff
@@ -226,6 +281,21 @@
         }
       ],
       "properties": {
+        "kind": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "The kind of the serviceGroup."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The serviceGroup tags."
+        },
         "properties": {
```

Comment:
[ARMBlockingComment] Proxy resource do not support tags. please remove

Reply:
That's correct. But `serviceGroups` is a new resource container ARM is introducing in the platform and ARM is adding a support for tags. I will follow up with you.